methods.rb

```diff
@@ -0,0 +1,171 @@
+# rubocop:disable all"
+...
```

Comment:
Line number 2 gives an error "unexpected ..."

Reply:
It works when you comment it out. 
The purpose of the line 2 was to disable all cops